### PR TITLE
docs: Remove helm installation guide and Add helm documentation link

### PR DESCRIPTION
**Pull request by qudtjs0753**
Fixes #TODO

### Motivation
- In [configure artifact-repository](https://argoproj.github.io/argo-workflows/configure-artifact-repository/), Helm installation   for mac is in this documentation. 
image

I think it will be better remove this description and add link to [helm documentation.](https://helm.sh/docs/intro/install/).

### Modifications
Changed documentation is like this

image

### CODE REVIEW

File: docs/configure-artifact-repository.md [hunk 1]
```diff
@@ -25,12 +25,7 @@ The actual repository used by a workflow is chosen by the following rules:
 
 ## Configuring MinIO
 
-```bash
-brew install helm # mac, helm 3.x
-helm repo add minio https://helm.min.io/ # official minio Helm charts
-helm repo update
-helm install argo-artifacts minio/minio --set service.type=LoadBalancer --set fullnameOverride=argo-artifacts
```
Comment (agilgur5) [inline docs/configure-artifact-repository.md:32]: you'd still need these 3 commands to install Minio.

File: docs/configure-artifact-repository.md [hunk 2]
```diff
@@ -25,12 +25,15 @@ The actual repository used by a workflow is chosen by the following rules:
...
+
+First, [install `helm`](https://helm.sh/docs/intro/install/). Then, install MinIO with the below commands:
+
 ```bash
-brew install helm # mac, helm 3.x
 helm repo add minio https://helm.min.io/ # official minio Helm charts
 helm repo update
 helm install argo-artifacts minio/minio --set service.type=LoadBalancer --set fullnameOverride=argo-artifacts
-```
+\```
```
Comment (agilgur5) [inline docs/configure-artifact-repository.md:36]: GitHub can't render code blocks in code blocks, so the `\` escape ends up visible. Could you manually remove the `\`?

Comment (qudtjs0753) [reply docs/configure-artifact-repository.md]: Sorry.. I should have reviewed your proposal, but I couldn't. I'll fix it right away

Comment (qudtjs0753) [reply docs/configure-artifact-repository.md]: The result is as follows
image

**Review by agilgur5 — APPROVED** [review]
Thanks for the contribution!

File: docs/configure-artifact-repository.md (hunk 2 above)
Comment (agilgur5) [reply docs/configure-artifact-repository.md]: > Sorry.. I should have reviewed your proposal, but I couldn't. I'll fix it right away

no worries, it's a GitHub issue unfortunately that pops up often with docs code blocks

> The result is as follows

LGTM!

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**